### PR TITLE
Adds keyboard suikagiken_suika85ergo

### PR DESCRIPTION
**Pull request by suikagiken**
## Description

Adds suikagiken suika85ergo as a new keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* n/a 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/suikagiken/config.h
Comment (waffle87) [inline keyboards/suikagiken/config.h:1]: This file should be removed.

File: keyboards/suikagiken/keymaps/default/config.h
Comment (waffle87) [inline keyboards/suikagiken/keymaps/default/config.h:1]: This file should be removed.

File: keyboards/suikagiken/info.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        "cols": ["B3", "B2", "B6", "F0", "F1", "C7", "D5", "B7", "B5", "B4", "E6", "D7", "C6", "D4", "D0", "D1", "D2"],
+        "rows": ["B0", "F4", "F5", "F6", "F7", "B1"]
+    },
+    "processor": "atmega32u4",
+    "url": "https://github.com/suikagiken/suika85ergo",
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x4B48",
+        "pid": "0x0002"
+    },
```
Comment (waffle87) [inline keyboards/suikagiken/info.json:25]:
```suggestion
    },
    "dynamic_keymap": {
        "layer_count": 2
    },
```

File: keyboards/suikagiken/keymaps/default/rules.mk
Comment (waffle87) [inline keyboards/suikagiken/keymaps/default/rules.mk:1]: This file should be removed. VIA is not to be enabled in the `default` keymap, it should be in it's own `via` keymap directory.

File: keyboards/suikagiken/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                          KC_LCTL, KC_LWIN, KC_LOPT, MO(1)  , KC_SPC , KC_LEFT, KC_DOWN, KC_RGHT, KC_SPC , MO(1)  , KC_ROPT, KC_APP , KC_RCTL  
+    ),
+    [1] = LAYOUT_6x17(
+        _______, _______, _______, _______, _______, _______, _______,                            _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______,          _______,          _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______,          _______,          _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______,  
+                 _______, _______, _______, _______, _______, _______,          KC_PGUP,          _______, _______, _______, _______, _______, _______,  
+                          _______, _______, _______, MO(1)  , _______, KC_HOME, KC_PGDN, KC_END , _______, MO(1)  , _______, _______, _______
```
Comment (waffle87) [inline keyboards/suikagiken/keymaps/default/keymap.c:21]:
```suggestion
                          _______, _______, _______, _______  , _______, KC_HOME, KC_PGDN, KC_END , _______, _______  , _______, _______, _______  
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by suikagiken** [comment]
@waffle87 @lesshonor 
Thank you for your suggestion. I have made the changes and committed them, please check back.

File: keyboards/suikagiken/suika85ergo/info.json [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    },
+    "dynamic_keymap": {
+        "layer_count": 2
+    },
+    "build": {
+        "debounce_type": "sym_eager_pk"
+    },
+    "debounce": 50,
+    "layouts": {
+        "LAYOUT_6x17": {
```
Comment (waffle87) [inline keyboards/suikagiken/suika85ergo/info.json:34]:
```suggestion
        "LAYOUT": {
```

File: keyboards/suikagiken/suika85ergo/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_6x17(
+        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  ,                            KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 , KC_DEL , 
+        C(KC_X), KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   ,          KC_PSCR,          KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_BSPC, 
+        C(KC_C), KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   ,          KC_EQL ,          KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_BSLS, KC_ENT , 
+        C(KC_V), KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_LBRC,          KC_RBRC, KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,  
+                 KC_LSFT, KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   ,          KC_UP  ,          KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,  
+                          KC_LCTL, KC_LWIN, KC_LOPT, MO(1)  , KC_SPC , KC_LEFT, KC_DOWN, KC_RGHT, KC_SPC , MO(1)  , KC_ROPT, KC_APP , KC_RCTL  
+    ),
+    [1] = LAYOUT_6x17(
```
Comment (waffle87) [inline keyboards/suikagiken/suika85ergo/keymaps/default/keymap.c:15]:
```suggestion
    [0] = LAYOUT(
        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  ,                            KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 , KC_DEL , 
        C(KC_X), KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   ,          KC_PSCR,          KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_BSPC, 
        C(KC_C), KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   ,          KC_EQL ,          KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_BSLS, KC_ENT , 
        C(KC_V), KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_LBRC,          KC_RBRC, KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,  
                 KC_LSFT, KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   ,          KC_UP  ,          KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,  
                          KC_LCTL, KC_LWIN, KC_LOPT, MO(1)  , KC_SPC , KC_LEFT, KC_DOWN, KC_RGHT, KC_SPC , MO(1)  , KC_ROPT, KC_APP , KC_RCTL  
    ),
    [1] = LAYOUT(
```

File: keyboards/suikagiken/suika85ergo/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_6x17(
+        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  ,                            KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 , KC_DEL , 
+        C(KC_X), KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   ,          KC_PSCR,          KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_BSPC, 
+        C(KC_C), KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   ,          KC_EQL ,          KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_BSLS, KC_ENT , 
+        C(KC_V), KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_LBRC,          KC_RBRC, KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,  
+                 KC_LSFT, KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   ,          KC_UP  ,          KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,  
+                          KC_LCTL, KC_LWIN, KC_LOPT, MO(1)  , KC_SPC , KC_LEFT, KC_DOWN, KC_RGHT, KC_SPC , MO(1)  , KC_ROPT, KC_APP , KC_RCTL  
+    ),
+    [1] = LAYOUT_6x17(
```
Comment (waffle87) [inline keyboards/suikagiken/suika85ergo/keymaps/via/keymap.c:15]:
```suggestion
    [0] = LAYOUT(
        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  ,                            KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 , KC_DEL , 
        C(KC_X), KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   ,          KC_PSCR,          KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_BSPC, 
        C(KC_C), KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   ,          KC_EQL ,          KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_BSLS, KC_ENT , 
        C(KC_V), KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_LBRC,          KC_RBRC, KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,  
                 KC_LSFT, KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   ,          KC_UP  ,          KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,  
                          KC_LCTL, KC_LWIN, KC_LOPT, MO(1)  , KC_SPC , KC_LEFT, KC_DOWN, KC_RGHT, KC_SPC , MO(1)  , KC_ROPT, KC_APP , KC_RCTL  
    ),
    [1] = LAYOUT(
```

File: keyboards/suikagiken/suika85ergo/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        _______, _______, _______, _______, _______, _______, _______,                            _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______,          _______,          _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______,          _______,          _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______,  
+                 _______, _______, _______, _______, _______, _______,          KC_PGUP,          _______, _______, _______, _______, _______, _______,  
+                          _______, _______, _______, MO(1)  , _______, KC_HOME, KC_PGDN, KC_END , _______, MO(1)  , _______, _______, _______
```
Comment (waffle87) [inline keyboards/suikagiken/suika85ergo/keymaps/via/keymap.c:21]:
```suggestion
                          _______, _______, _______, _______, _______, KC_HOME, KC_PGDN, KC_END , _______, _______, _______, _______, _______  
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by suikagiken** [comment]
As pointed out, Elite-C is used. I've also fixed the other points, thank you.

**Review by waffle87 — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**